### PR TITLE
get cv using query parameter

### PR DESCRIPTION
**Pull request by kamalrsa**
# Description

Get_api/cv endpoint using a query parameter 

Fixes # 5

# How to test?

cvbytitle
cvget

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by CharlesKimMaina — COMMENTED** [review]
It looks great Kamal, especially .where('title', 'like', `%${title}%`);,.,. i checked it n its working great,

**Review by benna100 — APPROVED** [review]

**Review by CharlesKimMaina — COMMENTED** [review]
It looks great Kamal, especially .where('title', 'like', %${title}%);,.,. i checked it n its working great,

File: src/server/api/controllers/cvs.controller.js [hunk 1]
```diff
@@ -4,18 +4,30 @@ const knex = require('../../config/db');
...
 
-const getCvs = async () => {
+const getCvs = async (title, limit) => {
   try {
-    return await knex('cv').select('cv.id', 'cv.title');
+    if (title) {
+      return await knex('cvs')
+        .select('*')
+        .where('title', 'like', `%${title}%`);
+    } else if (limit) {
```
Comment (pdsorensen) [inline src/server/api/controllers/cvs.controller.js:13]: @kamalrsa can i have you fix this one? :)

**Review by pdsorensen — COMMENTED** [review]
Only thing, and its ready to get merged!

**Comment by kamalrsa** [comment]
I fixed those unnecessary else error

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**